specification/dimensions/resourcetype.md

```diff
@@ -0,0 +1,53 @@
+# Resource Type
+
+A Resource ID is an identifier assigned to a resource by the provider. The Resource ID is commonly used for cost reporting, analysis, and allocation scenarios.  While Resource ID and Resource Name identifies a unique instance of a resource, the Resource Type describes the identity of the resource. Resource Type is commonly used for cost reporting, analysis, and allocation scenarios.
```

Comment:
nit: Do we need to speak to resource ID and name? We can probably jump right into Resource Type, right?

The resource type isn't really about the identity of the resource. It's more about the kind of resource it is. I added what Bing Chat says below. Feel free to take it or leave it :-) 

For scenarios, we should probably be more specific since everything can be used for reporting and analysis.
```suggestion
A Resource Type is a logical representation of a specific kind of cloud service or resource that can be provisioned, configured, and managed by users. Resource Type is commonly used for scenarios like identifying changes in cost over time or building a resource inventory.
```

Reply:
I like the rewrite but I also like demonstrating the juxtaposition of the Resource Type vs. the Resource ID/Resource Name.

How about?

```
A Resource Type is a logical representation of a specific kind of cloud service or resource that can be provisioned, configured, and managed by users. Resource Type is commonly used for scenarios like identifying changes in cost over time or building a resource inventory.  While Resource ID and Resource Name identifies a unique instance of a resource, the Resource Type describes the identity of the resource.
```

Reply:
WOrks for me. My only comment would be the "type" isn't really an "identity" of the resource.